config/dev.exs

```diff
@@ -92,7 +92,10 @@ config :trento, Trento.Scheduler,
   ]
 
 config :trento, Trento.Integration.Telemetry, adapter: Trento.Integration.Telemetry.ToLogger
-config :trento, Trento.Integration.Checks, adapter: Trento.Integration.Checks.MockRunner
+
+config :trento, Trento.Integration.Checks,
+  adapter: Trento.Integration.Checks.MockRunner,
+  runner_url: ""
```

Comment:
the `runner_url` is configuration owned by the `Trento.Integration.Checks.Runner` adapter